stellar-contract-env-host/src/test.rs

```diff
@@ -1,22 +1,27 @@
 use crate::{
     xdr::{
-        LedgerEntryData, LedgerKey, LedgerKeyContractData, ScHostFnErrorCode, ScHostObjErrorCode,
-        ScObject, ScObjectType, ScStatic, ScStatus, ScVal, ScVec,
+        ScHostFnErrorCode, ScHostObjErrorCode, ScObject, ScObjectType, ScStatic, ScStatus, ScVal,
+        ScVec,
     },
     Host, HostError, IntoEnvVal, Object, RawVal, Tag,
 };
 
-use crate::xdr;
-
-use im_rc::OrdMap;
 use stellar_contract_env_common::{
-    xdr::{ScMap, ScMapEntry, WriteXdr},
+    xdr::{ScMap, ScMapEntry},
     CheckedEnv, RawValConvertible,
 };
 
-use crate::storage::{AccessType, Footprint, Storage};
 use hex::FromHex;
 
+#[cfg(any(feature = "vm", feature = "testutils"))]
```

Comment:
Why do we need testutils mentioned here? If we're in a test, presumedly the testutils feature is always enabled?

Reply:
That isn't true at the moment. That would require putting a `testutils` conditional around the test module.

Reply:
So most of the test cases will be compiled out even though they don't use `testutils`.